cmd/agent/main.go

```diff
@@ -81,7 +81,8 @@ func main() {
 				log.Printf("[WARN] [main,podman] [message: Unable to retrieve local agent IP address, using '%s' instead] [error: %s]", options.AgentServerAddr, err)
 				advertiseAddr = options.AgentServerAddr
 			} else {
-				log.Fatalf("[ERROR] [main,docker] [message: Unable to retrieve local agent IP address] [error: %s]", err)
+				log.Printf("[ERROR] [main,docker] [message: Unable to retrieve local agent IP address] [error: %s]", err)
+				advertiseAddr = options.AgentServerAddr
```

Comment:
I think this should be in a different PR - I'm playing with non-container agent use to make my mTLS thing easier to play with

Reply:
is it in the other PR, if so, should this be reverted?

Reply:
Probaby should be in another PR

Reply:
I've relocated this in https://github.com/portainer/agent/pull/239